kurtosis-devnet/pectra.yaml

```diff
@@ -2,7 +2,7 @@ optimism_package:
   chains:
     - participants:
       - el_type: op-geth
-        el_image: ""
+        el_image: us-docker.pkg.dev/oplabs-tools-artifacts/images/op-geth:v1.101500.0-rc.3
```

Comment:
We should add an op-reth/op-node pair to this devnet and rerun the tests.

Reply:
Confirmed this works on commit [39d59be](https://github.com/ethereum-optimism/optimism/pull/14046/commits/39d59becb618dd85ee328a50884cf506e679ff9c)